build/README.md

```diff
@@ -5,6 +5,7 @@ Building Kubernetes is easy if you take advantage of the containerized build env
 ## Requirements
 
 1. Docker, using one of the following configurations:
+  1. **Minikube on Mac/Windows/Linux** you can use [minikube](https://github.com/kubernetes/minikube) to get a Docker build endpoint. Ensure you have at least 3GB of memory. See instructions for [setting up minikube for a build](#minikube).
```

Comment:
The default memory is 2G:
"DefaultMemory       = 2048" in file https://github.com/kubernetes/minikube/blob/master/pkg/minikube/constants/constants.go